implementations/rust/ockam/ockam_identity/src/identities/identities_creation.rs

```diff
@@ -182,7 +197,7 @@ impl IdentitiesCreation {
     pub async fn update_identity(&self, identity: &Identity) -> Result<()> {
```

Comment:
Do you think this function should be moved to the Repository implementation itself, and be wrapped into a transaction?

Reply:
The problem here is that Repository would need to depend on a VerifyingVault to do that...